packages/mailer/src/Email.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tempest\Mail;
+
+interface Email
+{
+    /**
+     * The envelope of the email.
+     */
+    public Envelope $envelope {
```

Comment:
Referring back to my previous question: I'd rather have the properties encapsulated by `Envelope` and `Content` directly on the email instead